src/context/UserContext/UserContext.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, createContext, useContext } from 'react';
+
+export const defaultValue = {
+    isAttestant: false,
+    setIsAttestant: () => undefined,
+};
+type UserContext = {
+    isAttestant: boolean;
+    setIsAttestant: React.Dispatch<React.SetStateAction<UserContext>>;
+};
+const UserContext = createContext<UserContext>(defaultValue);
```

Comment:
Trenger vel ikke å eksponere default-verdien.
```suggestion
type UserContextState = {
    isAttestant: boolean;
    setIsAttestant: React.Dispatch<React.SetStateAction<UserContext>>;
};
const UserContext = createContext<UserContextState>({
    isAttestant: false,
    setIsAttestant: () => undefined,
});
```